eva-accession-release-automation/gather_clustering_counts/requirements.txt

```diff
@@ -1 +1,2 @@
 ebi_eva_common_pyutils
+pytz
```

Comment:
Will be able to use the builtin [zoneinfo](https://docs.python.org/3/library/zoneinfo.html) module instead?

Reply:
Yes, we're on 3.8 in the cluster which is indeed why I didn't use zoneinfo...  We can use one of the other suggestions, I'm curious what's the objection to `pytz` though?  `backports` and `dateutil` also require an additional dependency.

Reply:
#optional I was just suggesting that just to have one less external dependency.